main.js

```diff
@@ -0,0 +1,69 @@
+var sum = items.reduce(function(a,b){
+  return a + b.price
+}, 0)
+
+var avg = sum/items.length
+
+document.querySelector("#answer1").innerHTML = "$" + avg.toFixed(2)
+
+var buget = items.filter(function(item){
+  if(item.price > 14 && item.price < 18){
+    return item.title
+  }
+})
+
+var bugetToString = ""
+buget.forEach(function(item){
+  bugetToString += item.title + "\n"
+})
+
+document.querySelector("#answer2").innerHTML = bugetToString
+
+var hasCode = items.filter(function(item){
+  return item.currency_code === "GBP"
+})
+
+var strCode = hasCode[0].title
+var strCode2 = hasCode[0].price
+
+document.querySelector("#answer3").innerHTML = strCode + " " + "&#163;" + strCode2
+
+var isWood = items.filter(function(item){
```

Comment:
I have no idea what you are trying to do here.  Why would you not just use the filter function to return a list of items that are wood then print that out below?